ibm/mas_devops/roles/ocp_config/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # Configuration Settings
 # -----------------------------------------------------------------------------
 ocp_ingress_update_timeouts: "{{ lookup('env', 'OCP_INGRESS_UPDATE_TIMEOUTS') | default('False', true) | bool }}"
-ocp_apiserver_update_ciphers: "{{ lookup('env', 'OCP_APISERVER_UPDATE_CIPHERS') | default('False', true) | bool }}"
+ocp_update_ciphers_for_semeru: "{{ lookup('env', 'OCP_UPDATE_CIPHERS_FOR_SEMERU') | default('False', true) | bool }}"
```

Comment:
As the `OCP_APISERVER_UPDATE_CIPHERS` is already in a release we need to either continue to support it (but deprecate it and also add OCP_UPDATE_CIPHERS_FOR_SEMERU that sets the same thing) or just make this a major change.

Reply:
I've updated the PR title and pushed a major commit. 

Also tested one more time with a Fyre cluster configured for FIPS.